common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -521,6 +521,8 @@ function (HTML5Video, Resizer) {
         if (this.videoPlayer.isPlaying()) {
             this.videoPlayer.runTimer();
         }
+        // Update the the current time when user seek. (YoutubePlayer)
+        this.videoPlayer.currentTime = time;
```

Comment:
I wasn't able to find any references to `this.videoPlayer.currentTime` in any of the calls from this method. So I think we should either unnest line 497 or keep this. The logic of this method is already complicated enough that introducing these duplicate calls would make it even harder for the next person to understand it.

Can you try removing 497 or removing this and un-nesting 497?